plugins/resteasy-http/src/main/java/org/radargun/http/service/RESTEasyCacheInfo.java

```diff
@@ -74,11 +77,11 @@ public long getTotalSize() {
                size = Long.parseLong(responseStr);    //if the call was done to nodejs app, then the long size is returned
 
             } catch (NumberFormatException ex) {
-               //If the call was done to Infinispan Server, then the response is the list of all entries
-               //WARNING: As this way the REST API retrieves all keys from the cache and returns them as a response,
-               //this part of block may have an impact on the performance of the stage which rely on this trait.
-               String[] entries = responseStr.split(System.getProperty("line.separator"));
-               size = entries.length;
+               Matcher matcher = Pattern.compile("\"size\":(\\d+)").matcher(responseStr);
+               if (matcher.find()) {
+                  // size:100
```

Comment:
Is this comment an example of what size can be returned or is 100 magic number that isn't clear by default?

Reply:
Removed the comment